api/src/main/java/org/openmrs/module/appointments/util/PatientUtil.java

```diff
@@ -0,0 +1,52 @@
+package org.openmrs.module.appointments.util;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import org.openmrs.Patient;
+import org.openmrs.PersonAttribute;
+
+public class PatientUtil {
+
+    private static final String IDENTIFIER_NAME = "identifierName";
+    private static final String IDENTIFIER = "identifier";
+    private static final String TELEPHONE_CONTACT = "Telephone contact";
```

Comment:
Not really sure where this is used, but "Telephone contact" is not suitably generic and not found in all implementations by this name.  We should find a better way to make this match the necessary use cases generically.